doc/source/dev/roadmap-detailed.rst

```diff
@@ -427,7 +427,12 @@ The following improvements will help SciPy better serve this role.
     - nested ANOVA
     - analysis of covariance (ANCOVA)
 
-- Add additional tools for meta-analysis; currently we have just `combine_pvalues`.
+  Also, provide an infrastructure for implementing hypothesis tests.
+- Add additional tools for meta-analysis
+- Add tools for survival analysis
+- Replace inverse transform sampling with a more suitable method of random
+  variate sampling where appropriate
```

Comment:
@chrisb83 @tirthasheshpatel feel free to suggest alternative phrasing

Reply:
Could we write "Speed up random variate sampling (method `rvs`) of distributions by relying on the tools in stats.sampling where appropriate"?

(since inversion with `NumericalInversePolynomial` would still be the method of choice in most cases)

Reply:
@chrisb83's suggestion sounds good.

> since inversion with `NumericalInversePolynomial` would still be the method of choice in most cases

I think `TransformedDensityInversion` also works great for many (fat-tailed) distributions. We can compute numerical derivatives using something like:

```
def dpdf(self, x):
    h = max(x*1e-7, 1e-7)
    return (self.pdf(x+h) - self.pdf(x-h)) / (2*h)
```

Reply:
Thanks, I adopted a very similar phrasing.